sass/includes/_highlight-cards.scss

```diff
@@ -3,18 +3,13 @@
     $root: &;
     font-family: $font__supria-sans;
 
-    &__container {
-        padding: 3% 5%;
-
-        @media screen  and (min-width: map-get($grid-breakpoints, "sm")) {
-            padding: 3% 9%;
-            
-        }
-    }
```

Comment:
Note: You'll see this a couple of times, I'm moving this layout outside of the components here to be their own `body-container` component to allow greater consistency.